internal/redfish/redfish.go

```diff
@@ -90,7 +94,7 @@ func (c *APIClient) BoardInfo() (*api.Board, error) {
 func (c *APIClient) MachineUUID() (string, error) {
 	systems, err := c.Service.Systems()
 	if err != nil {
-		log.Printf("ignored system query err:%s\n", err.Error())
+		c.log.Errorw("ignore system query", "error", err.Error())
 		return "", err
```

Comment:
May you also remove the `return "", err` line and therewith rather use `c.log.Warnw`?
Although not critical yet there are chances that a valid system will be returned alongside an error that corresponds to another one

Reply:
But if UUID ist not accessible, this is considered fatal right ? Unsure though

Reply:
Ok, that's right I think. Then we should proper update the error message

Reply:
ok error message now bit more explicit.

Reply:
I mean we should not log `ignore system query` since we do not ignore it here

Reply:
Of course, stupid me. Now fixed.